monitoring/controllers/kube-prometheus-stack/namespace.yaml

```diff
@@ -4,3 +4,6 @@ metadata:
   name: monitoring
   labels:
     app.kubernetes.io/component: monitoring
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/warn: baseline
+    pod-security.kubernetes.io/audit: baseline
```

Comment:
These will issue fill the [audit log](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/) at every reconciliation, do we really need them?

Reply:
I thought that "yes". I don't think that it will hurt anyway. So ... do you recommend me to remove these annotations?

Reply:
> I don't think that it will hurt anyway. 

I think it does, storing the audit log is expensive, and with baseline in there we just inflate the cost and storage.

Does Talos work without them?